Client/Frontend/LoginManagement/Cells/LoginListTableViewCell.swift

```diff
@@ -108,6 +108,11 @@ class LoginListTableViewCell: ThemedTableViewCell {
         setConstraints()
     }
 
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        contentView.addSubview(contentStack)
```

Comment:
What is the thinking behind adding the contentStack here? It's already added in the init and from what I can see it is never removed.

Reply:
Hi @OrlaM the `contentStack` is removed in the `super.prepareForReuse()`

Reply:
So it is 😅

Reply:
🤣😅